src/System.Security.Cryptography.Cng/src/Resources/Strings.resx

```diff
@@ -159,6 +159,12 @@
   <data name="Cryptography_InvalidCipherMode" xml:space="preserve">
     <value>Specified cipher mode is not valid for this algorithm.</value>
   </data>
+  <data name="Cryptography_InvalidCurve" xml:space="preserve">
```

Comment:
This seems like CurveNotSupported is a better name.  "Invalid" to me sounds like "this G point doesn't meet the curve formula, I'm not sure what you're talking about".